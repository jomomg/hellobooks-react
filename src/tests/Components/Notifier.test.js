import React from "react";
import Notifier from "../../components/notifier";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("<Notifier/>", ()=>{
    test("it renders successfully", ()=>{
        const notifier = shallow(<Notifier/>);
        expect(shallowToJson(notifier)).toMatchSnapshot();
    });
});