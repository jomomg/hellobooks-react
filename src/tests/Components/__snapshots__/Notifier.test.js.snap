// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notifier/> it renders successfully 1`] = `
<div>
  <WithStyles(Snackbar)
    anchorOrigin={
      Object {
        "horizontal": "center",
        "vertical": "top",
      }
    }
    autoHideDuration={1000}
    onClose={[Function]}
    open={false}
  >
    <SnackbarContentWrapper
      message=""
      onClose={[Function]}
      variant=""
    />
  </WithStyles(Snackbar)>
</div>
`;
